Jenkinsfile

```diff
@@ -24,6 +24,22 @@ def workdir_linux = { String gopath ->
 	gopath + workpath_linux
 }
 
+properties([[$class: 'ParametersDefinitionProperty',
+	parameterDefinitions: [
+		[$class: 'StringParameterDefinition',
+			defaultValue: '',
+			description: 'controller-sdk-go commit sha to use in glide.yaml (not updated if empty)',
+			name : 'SDK_SHA'],
+		[$class: 'StringParameterDefinition',
+			defaultValue: '',
+			description: 'controller-sdk-go repo to use in glide.yaml (not updated if empty)',
+			name: 'SDK_GO_REPO'],
+		[$class: 'BooleanParameterDefinition',
+			defaultValue: true,
+			description: 'option to run e2e tests (default: true)',
+			name: 'RUN_E2E']
```

Comment:
@arschles they can be passed down from any upstream job that chooses to do so (for our intended case, they are passed down by the upstream `controller-sdk-go` job for PRs in the code [here](https://github.com/vdice/controller-sdk-go/blob/410d82be404ff97f72ee956bbba46e3918585f70/Jenkinsfile#L75-L83)).  If, however, no upstream job chooses to pass values for those, you can see the defaults values are empty strings (and thus the `glide.yaml` modification logic will not be executed).

That being said, I need to update this PR to remove the `RUN_E2E` -- I intend to PR this functionality separately.